app/views/modals/delete-resource.html

```diff
@@ -34,6 +34,14 @@
       </div>
 
       <div ng-switch="kind">
+        <div ng-switch-when="Deployment">
+          <strong>Note:</strong> None of the replica sets created by this deployment will be deleted.
+
+          To delete the deployment and all of its replica sets, you can run the command
+          <pre class="code prettyprint mar-top-md">oc delete deployment {{resourceName}} -n {{projectName}}</pre>
```

Comment:
did you check if that actually works, i don't know if kubectl/oc actually behaves like DCs/RCs in this regard